src/background/preferences.ts

```diff
@@ -26,6 +26,7 @@ export class Preferences {
     iconColor: 'default',
     isolation: {
       active: true,
+      autoEnableDelay: 0,
```

Comment:
Already better, thanks! Sorry for being nitpicky here, but personally I think that it still doesn't really describe what it does. Like, `autoEnable` could be read as "automatically enable after a delay", but it doesn't really have a relation to the `active` state being toggled. And you're right, the UI currently uses "enable/disable", but the code uses "active" - so, to stay consistent in the code I think it'd be good to use "active" everywhere (`setActiveState` does use it too). So `reactivateDelay` (and adjusting functions/vars accordingly) might be best after all - what do you think?

Reply:
Another alternative that came to mind: `automaticReactivateDelay` (personally I prefer spelling out things, like "automatic" vs "auto" in this case)

Reply:
No problem - what are your feelings on maintaining that prefix for the related methods in isolation.ts?  For now I'll just rename the delay var - but I'm happy to push the rename to the other related stuff too.

Reply:
...never mind.  I'm going to rename the internals to match.  It connects everything together better.